src/index.js

```diff
@@ -47,6 +47,12 @@ todos.forEach((todo) => {
       updateDescription(todo.index, e.target.value);
     }
   });
+  checkBox.addEventListener('change', () => {
+    const completed = checkTodo(todo.completed);
+    todo.completed = completed;
+    localStorage.setItem('todos', JSON.stringify(todos));
+    window.location.reload();
```

Comment:
- [ ] <b>OPTIONAL:</b> This not best practice considering what we working with here. there is no need to reload the page just so you can refresh the data on the screen. you could simply refresh DOM by the same function you are using when the page loads.

Reply:
Hello @rukundoeric! 

Thanks for providing me with the optional feedback. I am going to work on it!